ruby/lib/minitest/queue/junit_reporter.rb

```diff
@@ -79,7 +79,8 @@ def add_tests_to(testsuites, suite, tests)
             'classname' => suite,
             'assertions' => test.assertions,
             'time' => test.time,
-            'flaky_test' => test.flaked?
+            'flaky_test' => test.flaked?,
+            'run-command' => Minitest.run_command_for_runnable(test),
```

Comment:
Mixing hyphens and underscores on the keys here, not a huge thing just thought I’d call it out.

Reply:
`flaky_test` was the mistake here: underscores are not really common in XML compared to dashes.